samples/cpp/common/format_reader/MnistUbyte.cpp

```diff
@@ -2,12 +2,14 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <MnistUbyte.h>
-
 #include <fstream>
 #include <iostream>
 #include <string>
 
+// clang-format off
+#include "MnistUbyte.h"
+// clang-format on
```

Comment:
why do we need `clang-format off` ? I suppose if we use `""` for include files, it will be in this place in any case.

Reply:
I can change if needed, just did not know how scan for clang formatting implemented

Reply:
changed